packages/image/src/__test__/load.test.ts

```diff
@@ -0,0 +1,49 @@
+import { afterAll, beforeAll, describe, expect, it } from "vitest";
+import { type ImageSource, load } from "..";
+
+class MockImage {
+  public src = "";
+
+  public setAttribute(name: string, value: string) {
+    this[name] = value;
+  }
+
+  public constructor() {
+    MockImage.lastInstance = this;
+  }
+
+  public static lastInstance: MockImage;
+}
+
+describe("load function", () => {
+  const originalImage = window.Image;
+
+  beforeAll(() => {
+    window.Image = MockImage as unknown as typeof Image;
+  });
+
+  afterAll(() => {
+    window.Image = originalImage;
+  });
+
+  it("should load defaultSrc when webpSrc is not provided", () => {
+    const testImage: ImageSource = {
```

Comment:
Change this line please cause of same reason

Reply:
[13c98b1](https://github.com/fepack/image/pull/32/commits/13c98b125f7fb26afa848269ee87a916840e2dbe)